### PR TITLE
Admin approval

### PR DESCRIPTION
**Pull request by pivnicek**
Re: https://github.com/keboola/connection/pull/633

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -275,22 +277,34 @@ A Organization object has the following attributes:
...
-*Not yet implemented.*
-
 + Response 200
 
+    ```js
+        {
+          "id": 123,
+          "name": "sample org",
+          "created": "2015-10-02T11:03:44+0200",
+          "allowAutoJoin": "1",
```
Comment (Halama) [inline apiary.apib]: fuj :) it must be `"allowAutoJoin": true`

Comment (pivnicek) [inline apiary.apib]: :-)  fair comment!

**Comment by Halama** [comment]
the tests should cover also:
 - super admin or maintainer cannot join organization
 - super admin or mainteinar cannot change organization `autoJoin` flag
 - super admin or maintainer can join project belonging to ogranization with auto join turned on
 - super amdin or maintainer can request access to project belonging to organization - pending status

**Comment by pivnicek** [comment]
Right, they should indeed!  thanks

**Comment by pivnicek** [comment]
To test the maintainers rights, the maintainers members endpoints http://docs.keboolamanagementapi.apiary.io/#reference/maintainers/maintainer-members would need implementation.

File: tests/OrganizationsTest.php [hunk 2]
```diff
@@ -69,4 +69,137 @@ public function testOrganizationDetail()
...
+            $this->fail('User should be in list');
+        }
+
+        $this->client->removeUserFromOrganization($organization['id'], $foundUser['id']);
+
+        $admins = $this->client->listOrganizationUsers($organization['id']);
+        $this->assertCount(1, $admins);
+    }
+    
+    public function testOrganizationPermissions()
```
Comment (ondrejhlavacek) [inline tests/OrganizationsTest.php]: would probably be better to split into multiple functions, each covering one case only. it's testing too many things at once and in this approach if any assertion fails the remaining tests are skipped, which could lead to confusion or harder debug.

Comment (pivnicek) [inline tests/OrganizationsTest.php]: fair point, thanks.  It's a bad habit of mine.

**Comment by ondrejhlavacek** [comment]
@pivnicek can you please doublecheck `testSuperNoJoinOrganization`? I'm getting 
```
1) Keboola\ManageApiTest\OrganizationsTest::testSuperNoJoinOrganization
Keboola\ManageApi\ClientException: You don't have access to maintainer 1

/code/src/Client.php:446
/code/src/Client.php:400
/code/src/Client.php:127
/code/tests/OrganizationsTest.php:128

Caused by
GuzzleHttp\Exception\ClientException: Client error: `POST http://local.docker:8700/manage/maintainers/1/organizations` resulted in a `403 Forbidden` response:
{"error":"You don't have access to maintainer 1","exceptionId":"connection-devel-8b7a5ffd3fd4821b9cc43e65c6468bd3"}
```

all other tests pass, anything wrong on my end?

**Comment by pivnicek** [comment]
jo, sorry, I'll sort it.

**Pull request merged**